src/CommunityToolkit.Maui.Core/Interfaces/IGravatarImageSource.shared.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace CommunityToolkit.Maui.Core;
+
+/// <summary>Gravatar image source interface.</summary>
+public interface IGravatarImageSource : IStreamImageSource
+{
+	/// <summary>Gets a value indicating the control cache validity.</summary>
+	TimeSpan CacheValidity { get; }
+
+	/// <summary>Gets a value indicating whether the cache is enable.</summary>
+	bool CachingEnabled { get; }
+
+	/// <summary>Gets a value indicating the email.</summary>
+	string? Email { get; set; }
+
+	/// <summary>Gets a value indicating the default image.</summary>
+	DefaultImage Image { get; set; }
+
+	/// <summary>Gets a value indicating the Uri.</summary>
+	Uri? Uri { get; }
+}
+
+/// <summary>Default gravatar image enumerator.</summary>
+public enum DefaultImage
```

Comment:
Move it to Primitives folder

Reply:
Doing so by adding to GravatarImageSourceDefaults.shared.cs